cirras-underwriting-api/deploy-tools/server.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<Server port="-1" shutdown="SHUTDOWN">
+  <!-- TODO: are these worth parameterizing? -->
+  <Listener className="org.apache.catalina.core.JreMemoryLeakPreventionListener" />
+  <Listener className="org.apache.catalina.mbeans.GlobalResourcesLifecycleListener" />
+  <Listener className="org.apache.catalina.core.ThreadLocalLeakPreventionListener" />
+
+
+  <GlobalNamingResources>
+  </GlobalNamingResources>
+
+  <Service name="Catalina">
+    <Connector port="#{TOMCAT_PORT}#"
+      protocol="org.apache.coyote.http11.Http11NioProtocol"
+      maxThreads="#{MAX_THREADS}#"
+        connectionTimeout="#{CONNECTION_TIMEOUT}#"
+        compression="on"
+        compressionMinSize="#{COMPRESSION_MIN_SIZE}#"
+        noCompressionUserAgents="#{NO_COMPRESSION_USER_AGENTS}#"
+        compressibleMimeType="#{COMPRESSION_MIME_TYPES}#"
+        scheme="https"
+        sslImplementationName="org.apache.tomcat.util.net.openssl.OpenSSLImplementation"
+        secure="true"
+        clientAuth="false"
+        sslProtocol="TLS"
+        keystoreFile="/usr/lib/tomcat/conf/selfsigned.jks"
+        keystorePass="SECRET_KEY"
+        ciphers="TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA384_P521,
+                TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA384_P384,
+                TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA384_P256,
+                TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA_P521,
+                TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA_P384,
+                TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA_P256,
+                TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256_P521,
+                TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256_P384,
+                TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256_P256,
+                TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA_P521,
+                TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA_P384,
+                TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA_P256,
+                TLS_RSA_WITH_AES_256_GCM_SHA384,
+                TLS_RSA_WITH_AES_128_GCM_SHA256,
+                TLS_RSA_WITH_AES_256_CBC_SHA256,
+                TLS_RSA_WITH_AES_256_CBC_SHA,
+                TLS_RSA_WITH_AES_128_CBC_SHA256,
+                TLS_RSA_WITH_AES_128_CBC_SHA">
+           
+      <UpgradeProtocol className="org.apache.coyote.http2.Http2Protocol"
+        noCompressionStrongETag="true"/>
```

Comment:
Should this be removed, since it didn't fix the etag issue? I don't think we actually want to prevent compression of responses.

Reply:
I'll remove this, but I should note that noCompressionStrongETag is true by default in Tomcat 9, and hard-coded to always true in Tomcat 10

Reply:
Oh right, doesn't matter then 😄 .